CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 #### Upcoming Changes
 
+* Add _builtin suffix to builtin names e.g.: output -> output_builtin [#1005](https://github.com/lambdaclass/cairo-rs/pull/1005)
```

Comment:
It should mention it's (technically) a breaking change. Any code __not__ expecting the suffix would break.

Reply:
Done!
[4307125](https://github.com/lambdaclass/cairo-rs/pull/1005/commits/43071257e313440c1d272890badbbfaf08b439d8)